app/src/components/market/common/categories/index.tsx

```diff
@@ -46,10 +50,12 @@ const CategoryInput = styled(Textfield)<{ isSelected: boolean }>`
 `
 
 export const Categories = (props: Props) => {
-  const { categories, name, onChange, selectedCategory, ...restProps } = props
+  const { categories, first, loadMoreButton, name, onChange, selectedCategory, setFirst, ...restProps } = props
   const [selectedCustom, setSelectedCustom] = useState(false)
 
-  const allCategories = CATEGORIES.concat(categories.filter(item => CATEGORIES.indexOf(item) < 0))
+  let allCategories: string[]
+  categories.length > 0 ? (allCategories = categories) : (allCategories = CATEGORIES)
```

Comment:
This seems to be responsability of the component that use Categories, maybe passing categories as undefined and giving a default value here with `const { categories = CATEGORIES, ... } = props`. Or directly doing `categories={categories.length ? categories : CATEGORIES}`.